docs/source/detsim.rst

```diff
@@ -1,5 +1,299 @@
+certain volume (usually around 1x1x1 mm\ :math:`^3` voxels). Each voxel
+represents an energy deposition, a so-called *hit*, and is characterized
+by a 3D position :math:`\boldsymbol{r} = (x, y, z)`, the energy
+:math:`E` required to produce them, and the delay time :math:`t` from
+the start of the event (right figure). The
+collection of hits for a given even is saved, defining a *track* in the
+detector’s active volume. The fast-simulation is very fast compared with
+the full-simulation mode, on the :math:`\mu s` scale. However, since the
+simulation is stopped we do not know the signal that would be measured
+at the sensors.
+
+.. image:: images/detsim/fullsim_output.png
+   :width: 40%
+
+.. image:: images/detsim/fastsim_output.png
+   :width: 40%
+
+
+.. _Detsim input:
+
+Input
+-----
+
+Detsim input are NEXUS fast simulation tracks, these files must contain only the following non-empty information.
+
+ * ``/MC/hits``
+
+.. _Detsim output:
+
+Output
+------
+
+ * ``/MC/``: the data in the nexus input file is copied in the output file
+ * ``/Run/event_map``: mapping between nexus event id and IC event id. This mapping is needed because **detsim** might split nexus events into multiple IC events
+ * ``/Run/events``: IC event number and simulated timestamps based on **rate** configuration parameter
+ * ``/Run/runInfo``: run number of each event
+ * ``/pmtrd``: the PMT waveforms for each PMT
+ * ``/sipmrd``: the SiPM waveforms for each SiPM
+ * ``/Filters/active_hits``: nexus events without hits inside the active volume are filtered
+
+.. _Detsim config:
+
+Config
+------
+
+.. container::
+    :name: tab:detsim-parameters
+    
+    .. table:: List of **detsim** parameters used at each stage of the algorithm.
+
+      =================== ==========================================
+      \                   Parameter
+      =================== ==========================================
+      Electron simulation inverse ionization yield :math:`w_{io}`
+      \                   fano-factor :math:`F`
+      \                   drift velocity :math:`v_d`
+      \                   lifetime :math:`\tau`
+      \                   diffusion :math:`D_L`, :math:`D_T`
+      Photon simulation   inverse scintillation yield :math:`w_{sc}`
+      \                   EL gain :math:`G_{EL}`
+      \                   conde-policarpo factor :math:`J_{CP}`
+      Signal simulation   EL drift velocity :math:`v^{EL}_d`
+      \                   S1 Light Tables
+      \                   S2 Light Tables
+      \                   point spread function (PSF)
+      Waveform creation   pre-trigger time
+      \                   buffer length
+      \                   PMT waveforms bin width
+      \                   SiPM waveforms bin width
+      =================== ==========================================
+
+
+Workflow
+--------
+
+The city flow parts are shown in the diagram
+
+.. image:: images/detsim/workflow.png
```

Comment:
Add on top or below the diagram a list with the algorithms/parts that you describe below. Use :ref: to they can be linked below. You can check this to see how it is done: https://pr18-sw-docs.readthedocs.io/en/latest/irene.html#workflow

Reply:
I would also add a brief description here of the whole city and the connection between the different steps.